src/SmartTransactionsController.test.ts

```diff
@@ -359,12 +425,19 @@ describe('SmartTransactionsController', () => {
         'updateSmartTransactions',
       );
       expect(updateSmartTransactionsSpy).not.toHaveBeenCalled();
-      networkListener({ providerConfig: { chainId: '56' } } as NetworkState);
+      networkListener({
+        providerConfig: { chainId: '0x32', type: 'rpc', ticker: 'CET' },
+        selectedNetworkClientId: 'networkClientId',
+        networkConfigurations: {},
+        networksMetadata: {},
+      } as NetworkState);
       expect(updateSmartTransactionsSpy).not.toHaveBeenCalled();
     });
   });
 
   describe('updateSmartTransactions', () => {
+    // TODO rewrite this test... updateSmartTransactions is getting called via the checkPoll method which is called whenever state is updated.
```

Comment:
yeah... have run into issues like this as well when testing these polling controllers. Could try stubbing the prototype before instantiating the controller. Up to you if you want to address here as we found the test in this state